app/models/workers_scaling/algorithms/resources_usage_minimization.rb

```diff
@@ -68,53 +67,53 @@ def time_constraint_check(predicted, left)
     ##
     # Increases computational power to match required system throughput
     # Does nothing if there are starting Workers already
-    # Uses known infrastructures with highest throughput if limits are not reached
-    # If limits in all known infrastructures are reached, uses random unknown infrastructure
+    # Uses known configurations with highest throughput if limits are not reached
+    # If limits in all known configurations are reached, uses random unknown configuration
     def increase_computational_power
-      LOGGER.debug 'Need to increase computational power'
+      log(:debug, 'Need to increase computational power')
```

Comment:
Co do linijek - dochodzi jeszcze pusta linijka po logu.
Co do sensu logów - mile widziane wszystkie komentarze co do tego które zostawić a które nie, a przede wszystkim dlaczego. To, co jest w tym momencie, w większości zostało jeszcze z fazy wstępnej implementacji, więc definitywnie wymaga przemyślenia. Chciałbym za to usłyszeć trochę opinii na temat logów od kogoś innego niż ja i Michał, bo robimy to mocno "na oko".